tests/view/renderer.js

```diff
@@ -646,6 +646,80 @@ describe( 'Renderer', () => {
 			expect( domSelection.getRangeAt( 0 ).collapsed ).to.be.true;
 		} );
 
+		it( 'should add and remove inline filler in case <p>foo<b>[]bar</b></p>', () => {
```

Comment:
I feel that these two tests are too long. Can't they be split into more concrete parts?

Reply:
E.g. there's just one new test for the whole change with not removing the filler during composition while there are at least two things to check – whether the filler is not touched during composition and that it's removed when it can be removed.

I mean – I know there are some pretty long tests in this file already, but your tests are twice as long :D

Reply:
The test is super long, I think I tried to omit some initialization code repetition and merged two checks into one test... I will split it and introduce more test to cover other cases if needed.